content/altinn-studio/guides/contributing/add-new-component/_index.en.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Add a new component in Studio
+description: Add a new component from Apps to make it configurable in Studio
+weight: 20
+---
+
+## Add a new component in Studio
+
+If new components are introduced to the Apps, either by the Apps team or by external contributors, the component can be added as a configurable component in Studio. This means that the properties for the component when used in a form, can be edited directly in Studio, just as any other component. 
+
+### Step-by-step guide
+
+1. [Get json schema for the new component](#1-get-json-schema-for-the-new-component)
+2. [Add an icon for the component](#2-add-an-icon-for-the-component)
+3. [Add the new component in the list of Studio components](#3-add-the-new-component-in-the-list-of-studio-components)
+4. [Ensure potential other component updates are supported*](#4-ensure-potential-other-component-updates-are-supported)
+
+_*Only relevant if running the schema generation script in step 1._
+
+### 1. Get json schema for the new component
+In order to retrieve the json schemas that defines the component config there are different options:
+- Manually write the json schema based on the `config.ts` in the apps repo for the component.
+- Run the script in `frontend/scripts/`. The usage of the script is described in the README. In order for the script to work the component you wish to add must have been released by app-frontend.
```

Comment:
Can we specify which script to run here? There are multiple scripts available in that folder